test/publisher_test.js

```diff
@@ -37,6 +37,24 @@ suite('publisher_test.js', function() {
     CCBuilder: msg => [],
   };
 
+  const declarationConstant = {
+    exchange: 'egg-hatched',
+    name: 'eggHatched',
+    title: 'Egg Hatched',
+    description: 'an egg hatched',
+    schema: 'egg-hatched-message.yml',
+    routingKey: [{
+      name:           'eggId',
+      summary:        'Identifier that we use for testing',
+      constant: 'primary',
```

Comment:
plz 2 be consistent with whitespace (either all lined up or all one space..)

Reply:
oh, my fancy IDE betrays me! 🙀 will do!